dev/plugins/hu.elte.txtuml.api.model.execution/src/hu/elte/txtuml/api/model/execution/LockedSeqDiagExecutorException.java

```diff
@@ -0,0 +1,23 @@
+package hu.elte.txtuml.api.model.execution;
+
+import hu.elte.txtuml.api.model.Model;
+import hu.elte.txtuml.api.model.impl.SequenceDiagramRelated;
+
+/**
+ * Thrown by certain methods of the {@link SequenceDiagramExecutor} interface
+ * which cannot be called after that specific model executor has been started.
```

Comment:
Yes, I agree that writing a _sequence diagram_ here is clearer for a beginner user. Note however that the current version is right as well because a sequence diagram executor is a special model executor.

Reply:
Fixed.